src/common/utility.js

```diff
@@ -287,5 +287,12 @@ export const getVideoObject = (video_data) => {
     }
 }
 
+const getLimit = (input, limit) => {
+    if (input[limit - 1] === ' ') {
+        return limit - 1
+    }
+    return limit
+}
+
```

Comment:
Maybe you can leave a comment above explaining that you're removing the spaces before appending the `...`. Rest LGTM.